heartbeat/azure-events.in

```diff
@@ -0,0 +1,893 @@
+#!@PYTHON@ -tt
+#
+#	Resource agent for monitoring Azure Scheduled Events
+#
+# 	License:	GNU General Public License (GPL)
+#	(c) 2018 	Tobias Niekamp, Microsoft Corp.
+#				and Linux-HA contributors
+
+import os, sys, time, subprocess
+import json
+import urllib, urllib2, socket
```

Comment:
Regarding urllib here's an example of how it can be implemented for both Python 2 and 3:
https://github.com/ClusterLabs/fence-agents/blob/master/agents/gce/fence_gce.py#L8